app/js/components/common/savedResultsTable.js

```diff
@@ -28,11 +29,13 @@ const SavedResultsTable = ({
           </thead>
           <tbody>
             {
-              results.map((result, index) => {
-                const { uuid, description } = result;
-                return (
+              results.map((result) => {
+                if (result.name != 'appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments') {
```

Comment:
@paulupendo @wanjikum This result comes from the database, and also it cannot be deleted from the database via the frontend. I dont know exactly what it is but I know it is not a saved cohort.